profiles/minimal/password-auth

```diff
@@ -10,6 +10,8 @@ account     required                                     pam_faillock.so
 account     required                                     pam_unix.so
 
 password    requisite                                    pam_pwquality.so
+password    [default=1 ignore=ignore success=ok]         pam_localuser.so                                       {include if "with-pwhistory"}
```

Comment:
Unless I'm missing something, it will skip pam_pwhistory and continue with the next module. See man pam.conf for more information on error=N syntax if this is the part that was unclear.

Reply:
Ok, that makes sense. Thank you!